README.md

```diff
@@ -1,14 +1,21 @@
-treeherder
+Treeherder
 ==================
 [![Build Status](https://travis-ci.org/mozilla/treeherder.png?branch=master)](https://travis-ci.org/mozilla/treeherder)
 [![Python Package Status](https://pyup.io/repos/github/mozilla/treeherder/shield.svg)](https://pyup.io/repos/github/mozilla/treeherder/)
 [![Node dependencies Status](https://david-dm.org/mozilla/treeherder/status.svg)](https://david-dm.org/mozilla/treeherder)
 [![Node devDependencies Status](https://david-dm.org/mozilla/treeherder/dev-status.svg)](https://david-dm.org/mozilla/treeherder?type=dev)
 [![Documentation Status](https://readthedocs.org/projects/treeherder/badge/?version=latest)](https://treeherder.readthedocs.io/?badge=latest)
 
+TABLE OF CONTENTS
+===================
```

Comment:
For longer READMEs a table of contents is definitely very useful, however here it makes up about a third of the length of the README - so I think adding one regresses the readability rather than increasing it.